config/webpacker.yml

```diff
@@ -25,10 +25,10 @@ development:
   <<: *default
 
 # uncomment these to enable webpack:server
-#  dev_server:
-#    host: 0.0.0.0
-#    port: 8080
-#    https: false
+  dev_server:
```

Comment:
Can't uncomment this, with this enabled, you *need* the server to be running.

And it's a non-issue anyway, people who don't use ui-classic as a plugin definitely don't need to run webpack-dev-server.